src/generated/resources/assets/dungeonsdelight/lang/zh_cn.json

```
@@ -0,0 +1,402 @@
+{
+  "block.dungeonsdelight.cut_stained_scrap": "切制污秽残片",
+  "block.dungeonsdelight.cut_stained_scrap_slab": "切制污秽残片台阶",
+  "block.dungeonsdelight.cut_stained_scrap_stairs": "切制污秽残片楼梯",
+  "block.dungeonsdelight.dungeon_stove": "地牢炉灶",
+  "block.dungeonsdelight.embedded_eggs": "嵌入蛋",
+  "block.dungeonsdelight.glow_berry_gelatin_block": "发光浆果凝胶块",
+  "block.dungeonsdelight.guardian_angel_block": "守护天使",
+  "block.dungeonsdelight.heap_of_ancient_eggs": "远古蛋堆",
+  "block.dungeonsdelight.living_fire": "活焰",
```

```suggestion
  "block.dungeonsdelight.living_fire": "活火",
```
保持与灵魂火（Soul Fire）的一致性。